brownie/test/plugin.py

```diff
@@ -56,9 +56,8 @@ def pytest_addoption(parser):
         )
 
 
-def pytest_configure(config):
+def pytest_load_initial_conftests(early_config, parser, args):
```

Comment:
If you aren't using `early_config`, `parser`, `args`, no need to include them as inputs

Reply:
```suggestion
def pytest_load_initial_conftests():
```